editor/debugger/editor_profiler.cpp

```diff
@@ -402,14 +404,24 @@ void EditorProfiler::_clear_pressed() {
 	_update_plot();
 }
 
+void EditorProfiler::_autostart_pressed() {
+	autostart_button->release_focus();
+	ProjectSettings::get_singleton()->set("debug/settings/profiler/autostart_profiler", autostart_button->is_pressed());
```

Comment:
Use dedicated method instead of magic setter:

```suggestion
	ProjectSettings::get_singleton()->set_setting("debug/settings/profiler/autostart_profiler", autostart_button->is_pressed());
```